react/features/toolbox/components/native/styles.js

```diff
@@ -130,3 +130,61 @@ export const toolbarToggledButtonStyles = {
     iconStyle: styles.whiteToolbarButtonIcon,
     style: styles.whiteToolbarButton
 };
+
+/**
+ * Overrides to the standard styles that we apply to the chat button, as that
+ * behives slightly differently.
```

Comment:
*behaves. Could you also tell me in what way it behaves differently?

Reply:
The chat button turns blue when toggled (unread messages available) instead of that gray color other buttons have. This special style object here is needed because of the style override logic in ```Toolbox.js#_renderToolbar```. It took me a while to understand how that works :/